base16-manager

```diff
@@ -465,43 +525,75 @@ set_random() {
   set_theme "$(list_themes -R | head -n1 | awk "{print $1;}")"
 }
 
+
+set_rofi() {
+  local package=$1
+  local theme=$2
+
+  # get template and handling for inexisting templates
+  if ls "$DATA_PATH/$package"/*/"base16-$theme."* &>/dev/null; then
+    file=$(ls "$DATA_PATH/$package"/*/"base16-$theme."*"rasi" )
+  else
+    err "$package: theme $theme not found."
+    return 1
+  fi
+
+  # path to rofi config
+  local ROFI_CONFIG="${XDG_CONFIG_HOME:-$HOME/.config}/rofi/config"
```

Comment:
I'm just not sure: The config is not stored as `config.rasi`, right?

Reply:
in my version (1.5.1) its in `~/.config/rofi/config`